SchematicNeo4j/SchematicNeo4j.Tests/NodeKey/NodeKey_Drop_Tests.cs

```diff
@@ -106,21 +112,19 @@ public void Dispose()
             using (var session = driver.Session(o => o.WithDefaultAccessMode(AccessMode.Write)))
             {
                 if (GetConstraints("NODE KEY", "Person").Count() == 1)
-                    session.WriteTransaction(tx => tx.Run($"DROP {personConstraint}"));
+                    session.ExecuteWrite(tx => tx.Run($"DROP CONSTRAINT {personConstraintRecord.name}"));
             }
         }
 
         private List<IRecord> GetConstraints(string ofType, string forLabel)
         {
             using (var session = driver.Session(o => o.WithDefaultAccessMode(AccessMode.Read)))
             {
-                return session.ReadTransaction(tx => tx.Run(
-                    "CALL db.constraints() yield description WHERE description contains (':'+$typeLabel+' ') AND description contains $constraintType RETURN description",
+                return session.ExecuteRead(tx => tx.Run(
+                    "SHOW CONSTRAINTS YIELD createStatement WHERE createStatement contains (':`'+$typeLabel+'`') AND createStatement contains $constraintType RETURN createStatement",
                     new { typeLabel = forLabel, constraintType = ofType }

```

Comment:
Ahhh, I think I replaced `description` with `name` in the past - will double check that and change to `createStatement`, thanks!

Reply:
@ttemple-Calibre using "createStatement" more closely mimics existing description, and ensures the labels and properties all match. By doing mutations and checks by name only, we don't have a guarantee that the constraints or indexes are equal to each other.  

If it helps, using `name` was my first thought as well. It's an easy trap to step into, and only by considering the **unhappy** edge cases does it become clear.